client/src/components/ReviewsShowPage.js

```diff
@@ -1,27 +1,30 @@
-import React from "react";
+import React, { useState } from "react";
 import ErrorList from "./registration/components/layout/ErrorList";
 import NewReviewForm from "./NewReviewForm";
+import RatingTile from "./RatingTile";
 
-const ReviewsShowPage = ({ activityName, reviews, postReview, errors }) => {
+const ReviewsShowPage = ({ activityName, reviews, postReview, errors, activityId, user }) => {
+  const listOfReviews = reviews.map((review) => (
```

Comment:
please rename this file to `ReviewList`, as it is not a page, and its not a show for one specific review